packages/dx-react-demos/src/bootstrap3/featured-virtual-scrolling/demo.jsx

```diff
@@ -84,11 +86,11 @@ export default class Demo extends React.PureComponent {
           }}
         />
 
-        <TableHeaderRow allowSorting allowGrouping />
+        <TableHeaderRow allowSorting allowGrouping allowDragging />
```

Comment:
The allowGrouping property may confuse our customers. It is not evident that allowDragging will allow grouping by d'n'd. May be it is a good time to rename it to showGroupingButton?